pinot-core/src/main/java/org/apache/pinot/core/query/pruner/SegmentPrunerService.java

```diff
@@ -52,7 +52,7 @@ public SegmentPrunerService(SegmentPrunerConfig config) {
   public boolean prune(IndexSegment segment, ServerQueryRequest queryRequest) {
     for (SegmentPruner segmentPruner : _segmentPruners) {
       if (segmentPruner.prune(segment, queryRequest)) {
-        LOGGER.debug("Pruned segment: {}", segment.getSegmentName());
+        LOGGER.debug("{} pruned segment: {}", segmentPruner.getClass().getName(), segment.getSegmentName());
```

Comment:
LOGGER  can be configured to log the class name, line number, etc. so we should not be logging class name

Reply:
I'm trying to log the class name for each SegmentPruner class being called, not the main SegmentPrunerService class